Makefile

```diff
@@ -64,6 +65,17 @@ docker-run-redis:
 docker-run-basic-auth:
 	@docker run -i -t --rm -e BASICAUTH_USERNAME=admin -e BASICAUTH_PASSWORD=12345 -e PODIUM_REDIS_HOST=$(MYIP) -e PODIUM_REDIS_PORT=6379 -p 8080:80 podium
 
+compose-up-dependencies:
+	@sed "s%<<LOCAL_GO_MODCACHE>>%${LOCAL_GO_MODCACHE}%g" deployments/docker-compose-model.yaml > deployments/docker-compose.yaml
+	@docker-compose -f deployments/docker-compose.yaml up -d redis-node-0 redis-node-1 redis-node-2 redis-standalone initialize-cluster
+
+compose-test:
+	@sed "s%<<LOCAL_GO_MODCACHE>>%${LOCAL_GO_MODCACHE}%g" deployments/docker-compose-model.yaml > deployments/docker-compose.yaml
+	@docker-compose -f deployments/docker-compose.yaml up --build podium-test
+
+compose-down:
+	@docker-compose -f deployments/docker-compose.yaml down
+
```

Comment:
Instead of adding this here. Add as another target and add this target as a pre-requisite to `compose-up-dependencies`.

This way you can reuse this for the `compose-down` as well.

Something like:

```makefile
compose-setup:
	@sed "s%<<LOCAL_GO_MODCACHE>>%${LOCAL_GO_MODCACHE}%g" deployments/docker-compose-model.yaml > deployments/docker-compose.yaml

compose-up-dependencies: compose-setup
	@docker-compose -f deployments/docker-compose.yaml up -d redis-node-0 redis-node-1 redis-node-2 redis-standalone initialize-cluster

compose-test: compose-setup
	@docker-compose -f deployments/docker-compose.yaml up --build podium-test

compose-down: compose-setup
	@docker-compose -f deployments/docker-compose.yaml down
```